dsl/src/test/groovy/org/cristalise/dsl/test/persistency/outcome/SchemaBuilderSpecs.groovy

```diff
@@ -632,6 +654,31 @@ class SchemaBuilderSpecs extends Specification implements CristalTestSetup {
     }
 
 
+    def 'Field value of decimal type can be restricted by precision and scale'() {
+        expect:
+        SchemaTestBuilder.build('Test', 'TestData', 0) {
+            struct(name: 'TestData') {
+                field(name: 'efficiency', type: 'decimal', precision: 2, scale: 2)
```

Comment:
add a test where precision != scale, just to make sure they are copied tot he correct field